### PR TITLE
Add more tools using `gh` utility to access github API

### PR DESCRIPTION
**Pull request by blag**
I've been working on these tools, since I wasn't pleased with the existing tools.

These tools are buggy, unfinished, and poorly documented, but since they might be useful to other people I'm publishing them in this draft PR.

### CODE REVIEW

File: rerun-workflows.sh [hunk 1]
```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# A helper script to bootstrap a new StackStorm Exchange pack.
```
Comment (arm4b) [inline rerun-workflows.sh]: :+1:

**Review by arm4b — COMMENTED** [review]
Yeah, would be indeed nice including detailed description at the header for other files too.
This will help someone better navigating them.

**Comment by cognifloyd** [comment]
This is required for https://github.com/StackStorm/community/issues/63

**Review by cognifloyd — APPROVED** [review]

**Comment by cognifloyd** [comment]
I'm merging this. We can add more docs or enhancements in follow-up PRs.

**Pull request merged**